indico/modules/events/contributions/contrib_fields.py

```diff
@@ -48,6 +49,15 @@ class ContribField(BaseField):
     config_form_base = ContribFieldConfigForm
     common_settings = ('title', 'description', 'is_required', 'is_active')
 
+    def __init__(self, obj, management=True):
+        super(ContribField, self).__init__(obj)
+        self.management = management
+
+    def required_validator(self):
```

Comment:
I'm pretty sure this is missing an `@property` (and at the same time I'd use a ternary-style statement inside the method)